### PR TITLE
[dev-launcher] Added `android:exported="true"` to the launcher activity

### PR DESCRIPTION
**Pull request by wbroek**
# Why
Fixes https://github.com/expo/expo/issues/16366 not be able to publish to Google Play

# How

As mentioned in the [Android docs ](https://developer.android.com/about/versions/12/behavior-changes-12#exported) if an activity has intent filters you need to explicitly set the exported attributes

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [x ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ Package-specific changes not released in any SDK will be added here just before
 - Updated `firebase` from `8.2.3` to `8.10.0`. ([#14259](https://github.com/expo/expo/pull/14259) by [@IjzerenHein](https://github.com/IjzerenHein))
 - Updated `@react-native-community/netinfo` from `6.0.0` to `6.0.2`. ([#14329](https://github.com/expo/expo/pull/14329) by [@cruzach](https://github.com/cruzach))
 - Updated `react-native-webview` from `11.6.2` to `11.13.0`. ([#14301](https://github.com/expo/expo/pull/14301) by [@kudo](https://github.com/kudo))
-- Updated `react-native-lottie` from `4.0.2` to `4.0.3`. ([#14331](https://github.com/expo/expo/pull/14331) by [@cruzach](https://github.com/cruzach))
+- Updated `lottie-react-native` from `4.0.2` to `4.0.3`. ([#14331](https://github.com/expo/expo/pull/14331) by [@cruzach](https://github.com/cruzach))
```
Comment (lukmccall) [inline CHANGELOG.md:189]: Please get rid of those changes.

File: packages/expo-dev-launcher/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 
 ### 🐛 Bug fixes
 
+- Added `android:exported="true"` to the activity. Cause on Android 12 and higher it needs to [explicity decalred](https://developer.android.com/about/versions/12/behavior-changes-12#exported) . ([#16367](https://github.com/expo/expo/pull/16367) by [@wbroek](https://github.com/wbroek))
```
Comment (lukmccall) [inline packages/expo-dev-launcher/CHANGELOG.md:11]:
```suggestion
- Added `android:exported="true"` to the activity. Cause on Android 12 and higher it needs to [explicity decalred](https://developer.android.com/about/versions/12/behavior-changes-12#exported). ([#16367](https://github.com/expo/expo/pull/16367) by [@wbroek](https://github.com/wbroek))
```

**Review by lukmccall — COMMENTED** [review]
Thanks for your contribution 🏅
Looks good to me ✅

File: CHANGELOG.md (hunk 1 above)
Comment (wbroek) [reply CHANGELOG.md]: Sorry for that

File: packages/expo-dev-launcher/CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,7 @@
 
 ### 🐛 Bug fixes
 
+- Added `android:exported="true"` to the activity. Cause on Android 12 and higher it needs to [explicity decalred](https://developer.android.com/about/versions/12/behavior-changes-12#exported). ([#16367](https://github.com/expo/expo/pull/16367) by [@wbroek](https://github.com/wbroek))
```
Comment (lukmccall) [inline packages/expo-dev-launcher/CHANGELOG.md:11]:
```suggestion
- Added `android:exported="true"` to the activity, cause on Android 12 and higher it needs to [explicity declared](https://developer.android.com/about/versions/12/behavior-changes-12#exported). ([#16367](https://github.com/expo/expo/pull/16367) by [@wbroek](https://github.com/wbroek))
```

**Review by lukmccall — APPROVED** [review]

**Pull request merged**